src/vs/workbench/contrib/webview/browser/webviewFindWidget.ts

```diff
@@ -54,7 +54,7 @@ export class WebviewFindWidget extends SimpleFindWidget {
 		this._delegate.focus();
 	}
 
-	protected _onInputChanged(): boolean {
+	protected async _onInputChanged(): Promise<boolean> {
```

Comment:
I don't think we need to make this change since we're using `.then` instead of await in TerminalFindWidget